design/specs/column-extraction.md

```diff
@@ -8,87 +8,58 @@ editor: markdown
 
-    <img width="1469" alt="image" src="https://user-images.githubusercontent.com/845767/177546415-23b11ded-8987-4e69-ab1a-1decb9e773d3.png">
+### Link Column Name Convention
 
+To help users understand the role of the link column, the name of the link column will follow a convention that will be applied when users name the new table. The convention will be:
 
-5. **The user navigates to the new table**
+`table_name` `+` `Id`
 
-    The new table will be listed in the list of tables for the current schema. The user can modify the table, add new columns or rename it if needed.
+![image](/assets/design/specs/column-extraction/185142290-db3d6fc7-86b6-4a2d-8c23-7ed48104c8cc.png)
 
-    <img width="874" alt="image" src="https://user-images.githubusercontent.com/845767/177550009-30fcae81-2ae9-4539-ad85-5a0d1289790b.png">
+### Allowing the user to modify the original column selection
 
-# Unresolved Design Problems
+Under the `Columns to Extract` section, the user can extend the original column selection by selecting additional columns. The user can also remove columns from the selection.
 
-## Extracting a foreign key column to create a mapping table
-If a user had a foreign key referencing a publisher in a publications table and wanted to link to more than one publisher, could they select both the table's primary key and that foreign key and create a new table from both? Should there be a special action that does this without adding a foreign key in place of the columns?
+This component could be reused for all actions involving column subsets.
 
-## Suggesting columns to be extracted
-Offering suggestions for columns that can be extracted into new tables would improve the feature's discoverability and provide users with a clear path to a better data model.
+Additionally, the user can rename the columns that will be extracted in cases where the column name is not descriptive enough.
 
-## UI for Inspector Panels
-The inspector's UI is still in its early stages and should not be implemented without additional design work.
+## Finalizing the Extract Column Operation
 
-## Specify the table's name and foreign key column name
-WIP
-## Multiple column actions
-WIP
-## Allow Duplicates
-WIP
-## Selection
-WIP
-### Column vs. Cell Selection
-WIP
-### Row and Column Selection Consistency
-WIP
-## Inspector Modes
-WIP
-### Column
-WIP
-### Cell
-WIP
-### Row
-WIP
-## Keyboard Modifiers
-WIP
-### Touch-only devices
-WIP
-### Visual Cues
-WIP
-## Reordering Columns
-WIP
-## Cell-Only Selection
-WIP
\ No newline at end of file
+Once the user is happy with the changes, they can click on the `Extract Columns` button. The inspector panel will be toggled and the new link column will be selected in the original table. Under the `Columns` section, the user will see the new table listed as a link.
+
+![image](/assets/design/specs/column-extraction/185143860-f2995fc4-c767-4cce-aaa6-319ef27630ff.png)
```

Comment:
I don't think there's enough information under "Link Properties" in the table inspector to implement it. Can you please either add details or make a note that that will be covered in a different spec?